tutorials/using-cloud-vpn-with-checkpoint/index.md

```diff
@@ -0,0 +1,383 @@
+
+Create a new Cloud router to configure GCP for Site to Site VPN connectivity.
+
+In Google Cloud Platform Console, select **Networking** > **[Cloud Routers](https://console.cloud.google.com/interconnect/routers)** > **Create Router**.
+
+Enter the parameters as shown in the following table and click **Create**.
+
+|Parameter|Value|Description|
+|---------|-----------|-----|
+|Name|`gcp-to-cp-router`|Name of the cloud router.|
+|Description|           |Description of the cloud router.|
+|Network|`to-cp`|The GCP network the cloud router attaches to. Note: This is the network which manages route information.|
+|Region|`europe-west1`|The home region of the cloud router.Note: Make sure the cloud router is in the same region as the sub-networks it is connecting to.|
+|Google ASN|`65000`|The Autonomous System Number assigned to the cloud router. Use any unused private ASN (64512 - 65534, 4200000000 – 4294967294).|
+
+![alt_text](Image_8.PNG)
+
+In Google Cloud Platform Console, select **Networking** > **Interconnect** > **[VPN](https://console.cloud.google.com/interconnect/vpn)** >> **CREATE VPN CONNECTION**.
+
+**Google Compute Engine VPN gateway**
+
+|Parameter|Value|Description|
+|---------|-----|-----------|
+|Name|`gcp-to-cp-vpn`|Name of the VPN gateway|
+|Description|`VPN tunnel connection between GCP and Check Point Security Gateway`|Description of the VPN gateway|
+|Network|`to-cp`|The GCP network the VPN gateway attaches to **Note**: This network will get VPN connectivity|
+|Region|`europe-west1`|The home region of the VPN gateway **Note**: Make sure the VPN gateway is in the same region as the subnetworks it is connecting to.|
+|IP address|`cloud-ip(35.195.227.26)`|The static public IP address used by the VPN gateway. An existing, unused, static public IP address within the project can be assigned, or a new one created.|
+
+![alt_text](Image_9.PNG)
+
+**Tunnels**
+
+|Parameter|Value|Description|
+|---------|------|-----------|
+|Name|`gcp-to-cp-vpn`|Name of the VPN tunnel|
+|Description|`VPN tunnel connection between GCP and Check Point Security Gateway`|Description of the VPN tunnel|
+|Remote peer IP address|`199.203.248.181`|Public IP address of the on-premise VPN appliance used to connect to Cloud VPN.
+|IKE version|`IKEv2`|The IKE protocol version. You can select IKEv1 or IKEv2.|
+|Shared secret|`secret`|A shared secret for authentication by the VPN gateways. Configure the on-premise VPN gateway tunnel entry with the same shared secret.|
+|Routing options|`Dynamic(BGP)`|Cloud VPN supports multiple routing options for the exchange of route information between the VPN gateways. In this example, Cloud Router and BGP are configured.|
+|Cloud Router|`gcp-to-cp-router`|Select the Cloud router created previously.|
+|BGP session| |BGP sessions enable your cloud network and on-premise networks to dynamically exchange routes|
+
+![alt_text](Image_10.PNG)
+
+### BGP Sessions
```

Comment:
Remove BGP Sessions.
Add Step 4: Enter the parameters as shown in the following table for the BGP sessions: